lib/jazzy/doc_builder.rb

```diff
@@ -264,17 +264,35 @@ def self.color_for_coverage(coverage)
     def self.download_badge(coverage, options)
       return if options.hide_documentation_coverage || !options.download_badge
 
-      warn 'downloading coverage badge'
-      badge_url = 'https://img.shields.io/badge/documentation-' \
-        "#{coverage}%25-#{color_for_coverage(coverage)}.svg"
+      coverage_length = coverage.to_s.size.succ
+      percent_string_length = coverage_length * 80 + 10
+      percent_string_offset = coverage_length * 40 + 975
+      width = coverage_length * 8 + 104 
+      svg = <<-SVG.gsub(/^ {8}/, '')
+        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="#{width}" height="20">
+          <linearGradient id="b" x2="0" y2="100%">
+            <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+            <stop offset="1" stop-opacity=".1"/>
+          </linearGradient>
+          <clipPath id="a">
+            <rect width="#{width}" height="20" rx="3" fill="#fff"/>
+          </clipPath>
+          <g clip-path="url(#a)">
+            <path fill="#555" d="M0 0h93v20H0z"/>
+            <path fill="##{color_for_coverage(coverage)}" d="M93 0h#{coverage_length / 10 + 10}v20H93z"/>
```

Comment:
`coverage_length / 10 + 10` isn't exactly right. For example, this produced `0h10v20H93z` whereas shields.io produced `0h35v20H93z`.

Reply:
For what coverage?

Reply:
Ah you know what, I think that should be percent_string_length

Reply:
Alamofire, among others:

```diff
Bacon::Error: File comparison error `docs/badge.svg` for document_alamofire:
--- DIFF -----------------------------------------------------------------------
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="a"><rect width="128" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#a)"><path fill="#555" d="M0 0h93v20H0z"/><path fill="#97CA00" d="M93 0h35v20H93z"/><path fill="url(#b)" d="M0 0h128v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="110"><text x="475" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="830">documentation</text><text x="475" y="140" transform="scale(.1)" textLength="830">documentation</text><text x="1095" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="250">86%</text><text x="1095" y="140" transform="scale(.1)" textLength="250">86%</text></g> </svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128" height="20">
+  <linearGradient id="b" x2="0" y2="100%">
+    <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+    <stop offset="1" stop-opacity=".1"/>
+  </linearGradient>
+  <clipPath id="a">
+    <rect width="128" height="20" rx="3" fill="#fff"/>
+  </clipPath>
+  <g clip-path="url(#a)">
+    <path fill="#555" d="M0 0h93v20H0z"/>
+    <path fill="#97CA00" d="M93 0h10v20H93z"/>
+    <path fill="url(#b)" d="M0 0h128v20H0z"/>
+  </g>
+  <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="110">
+    <text x="475" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="830">documentation</text>
+    <text x="475" y="140" transform="scale(.1)" textLength="830">documentation</text>
+    <text x="1095" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="250">86%</text>
+    <text x="1095" y="140" transform="scale(.1)" textLength="250">86%</text>
+  </g>
+</svg>
```